HousingRepairsSchedulingApi.Tests/ControllersTests/AppointmentsControllerTests.cs

```diff
@@ -7,12 +7,14 @@ namespace HousingRepairsSchedulingApi.Tests.ControllersTests
 {
     using System;
     using Controllers;
+    using Domain;
     using UseCases;
 
     public class AppointmentsControllerTests : ControllerTests
     {
         private const string SorCode = "SOR Code";
         private const string LocationId = "locationId";
+        RepairDescription orderComments = new RepairDescription{Text = "something"};
```

Comment:
Add field access modifier
```suggestion
        private RepairDescription orderComments = new RepairDescription{Text = "something"};
```